sycl/test-e2e/syclcompat/math/math_vectorized.cpp

```diff
@@ -101,6 +146,27 @@ int main() {
                                                         0xFFF8FFFD);
   test_vectorized_unary<syclcompat::abs, uint32_t>(0xFFFBFFFD, 0x00050003);
   test_vectorized_sum_abs_diff<uint32_t>(0x00010002, 0x00040002, 0x00000003);
```

Comment:
While the `vectorized_binary` with `relu` is being tested through the `vectorized_ternary`, It could be good to have at least a case testing that the `relu` is working as intended with `vectorized_binary`.